### DISCUSSION TITLE
Error: C stack usage  15923920 is too close to the limit

### DISCUSSION THREAD

**Discussion by edambo** [brad-cannell/detect_fu_interviews_public#18] [Q&A]
I'm getting this error when I try to run the code for the self-report codebook:
![image](https://user-images.githubusercontent.com/126591971/230502601-dd90aa92-64e6-48a3-b11c-9b22940634e5.png)
From my internet search, the issue might either be related to a recursive function or the fact that the object is too large. The only solution I've been able to find is to run ulimit -s unlimited in the terminal, but this doesn't work for Windows. I've spent a lot of time trying to figure this out but I haven't been able to yet.

**Comment by mbcann01** [comment]
Hi @edambo ! Did you get this figured out yet?

**Reply by edambo** [reply]
@mbcann01 I think I've been able to figure out the issue. There were some typos.

**Comment by edambo** [comment]
No, not yet

**Comment by mbcann01** [comment]
Awesome!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
